gradle/dirExclusions.gradle

```diff
@@ -0,0 +1,16 @@
+idea {
```

Comment:
It's not completely clear to me what's in each file by the file name. For example, idea.gradle, or ide.gradle may make more since in this example.